tests/console/yast2_http.pm

```diff
@@ -102,7 +102,7 @@ sub run {
     assert_screen 'http_vitual_host_page';          # check wizard page (4/5)
     send_key 'alt-n';                               # go to http server wizard (5/5) --summary
                                                     # make sure that apache2 server got started when booting
-    if ((is_sle '<15') || (is_leap)) {
+    if (is_sle('<15') || is_leap('<15.0')) {
```

Comment:
It's alright to keep original version as we execute this test against latest products only. But if we want to be strict it should be `<15.1` then, as 15.0 will get these changes as updates only.